fontforgeexe/charinfo.c

```diff
@@ -930,8 +930,31 @@ return( false );
     }
 
     /* If we get this far, then we didn't find any errors */
-    for ( pst = sc->possub; pst!=NULL; pst=pst->next )
-	pst->ticked = pst->type==pst_lcaret;
```

Comment:
What was this doing before?

Reply:
Elements that aren't `ticked` will be modified or deleted by the code below. I assume this was there to save existing `pst_lcaret` entries but they never wound up being copied in the first place. Every `ticked` can be false now because the new code that copies the `lcaret`s sets `ticked` to true (and if there aren't any the entry can and should be deleted).